src/Chapter15/Listing15.27.ProjectionToAnAnonymousType.cs

```diff
@@ -4,16 +4,18 @@ namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter15.Listing15_27
     using System.Collections.Generic;
     using System.IO;
     using System.Linq;
-
+    #region INCLUDE
+    #region EXCLUDE
```

Comment:
Would it be simpler to just move the #region INCLUDE to just surround the inner section instead of the #exclude right next to an include?